docs/guide/core-widgets.md

```diff
@@ -239,12 +274,57 @@ The sizes available by default are:
 
 The final image size name is `original`, which delivers the original image file. This should be used carefully since it could be very large and slow to download.
 
+### Adding a placeholder image
+
+The image widget displays a placeholder image by default. To block this behavior, set the `placeholder` option to a value of `false`. The default image can be changed by saving the desired image in the `public` folder of your module, then passing the final build path for that file to the `placeholderUrl` option.
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  options: {
+    // for a file named 'placeholder.png' in the module public folder
+    placeholderUrl: '/modules/@apostrophecms/my-image-widget/placeholder.png'
+  }
+};
+
+```
+
+<template v-slot:caption>
+/modules/@apostrophecms/image-widget/index.js
+</template>
+</AposCodeBlock>
+
+::: note
+The path to the custom image is to where the file will be located in the final build. If you customize the image widget at the project level, this is the `my-image-widget` folder, **not** the `image-widget` folder.
```

Comment:
I think this is still a little confusing, I might think I have to make a `my-image-widget` folder myself somewhere. Please make clear that this is automatic.

Reply:
Changed.